### PR TITLE
Feature/sergeyro/css modules migration 7

### PR DESCRIPTION
**Pull request by SergeyRoyt**
- [ ] 1. Clickable
- [ ] 2. ColorPicker
- [ ] 3. ColorPickerContentComponent
- [ ] 4. ColorPickerItemComponent
- [ ] 5. Combobox
- [ ] 6. ComboboxCategory
- [ ] 7. ComboboxOption

### CODE REVIEW

File: src/components/Combobox/Combobox.jsx [hunk 1]
```diff
@@ -178,19 +188,32 @@ const Combobox = forwardRef(
...
         ref={mergedRef}
-        className={cx("combobox--wrapper", className, `size-${size}`, {
-          empty: !hasResults,
-          "sticky-category": stickyCategories
-        })}
+        className={cx(
+          styles.comboboxWrapper,
+          "combobox--wrapper",
+          className,
+          styles[`${camelCase("size-" + size)}`],
```
Comment (hadasfa) [inline src/components/Combobox/Combobox.jsx:195]: same camel case note

**Review by hadasfa — APPROVED** [review]
approving for not being a blocker. please fix camelCase function issue before merging

**Comment by SergeyRoyt** [comment]
Refactored

**Comment by SergeyRoyt** [comment]
Merged into [Vibe2.0](https://github.com/mondaycom/monday-ui-react-core/pull/885)

**Pull request closed without merging**